OnlineStore/store/src/main/java/RandomStorePopulator.java

```diff
@@ -1,34 +1,70 @@
 import com.github.javafaker.Faker;
-
-import java.lang.reflect.InvocationTargetException;
+import org.reflections.Reflections;
+import product.Category;
+import product.Product;
 import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 public class RandomStorePopulator {
 
-    public ArrayList<Product> populateStore(){
-        Faker faker = new Faker();
-        ArrayList<Product> products = new ArrayList<>();
-        for (int i = 0; i < 21; i++){
+    List<String> nameOfCategoryClasses = new ArrayList<>();
+    Reflections reflections = new Reflections("categories");
```

Comment:
this block of code (for getting categories via reflection) is better to place in constructor or separate method